lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1523,25 +1523,109 @@ spec = do
               txs2 <- listTransactions ctx w (Just te) (Just te) Nothing
               length <$> [txs1, txs2] `shouldSatisfy` all (== 0)
 
-    describe "TRANS_DELETE_01 - Single Output Transaction for " $ do
-        txDeleteForgetSingleOutputTxTest
-            "wallets" it fixtureWallet
-            postTxEp listTxEp deleteTxEp getWalletEp
-        -- xit -> it when submitting byron tx is supported by Jormungadr
-        -- Then we need also to add impl of fixtureByronWallet in DSL
-        txDeleteForgetSingleOutputTxTest
-            "byron-wallets" xit fixtureByronWallet
-            postByronTxEp listByronTxEp deleteByronTxEp getByronWalletEp
-
-    describe "TRANS_DELETE_02 - checking not pending anymore error for " $ do
-        txDeleteNoLongerPendingTest
-            "wallets" it fixtureWallet
-            postTxEp listTxEp deleteTxEp
-        -- xit -> it when submitting byron tx is supported by Jormungadr
-        -- Then we need also to add impl of fixtureByronWallet in DSL
-        txDeleteNoLongerPendingTest
-            "byron-wallets" xit fixtureByronWallet
-            postByronTxEp listByronTxEp deleteByronTxEp
+    it "TRANS_DELETE_01 -\
+        \ Shelley: Can forget pending transaction" $ \ctx -> do
```

Comment:
this test for Shelley is the same as before, I just `derived` it from the method since it is not reusable for Byron...